src/py/mat3ra/made/tools/modify.py

```diff
@@ -35,22 +34,33 @@ def filter_by_label(material: Material, label: Union[int, str]) -> Material:
     return new_material
 
 
-@decorator_convert_material_args_kwargs_to_structure
-def translate_to_bottom(structure: PymatgenStructure, use_conventional_cell: bool = True):
+def translate_atoms(
+    material: Material,
+    vector: Optional[List[float]] = None,
+    to: Optional[Literal["top", "bottom", "center"]] = None,
+) -> Material:
     """
-    Translate atoms to the bottom of the cell (vacuum on top) to allow for the correct consecutive interface generation.
-    If use_conventional_cell is passed, conventional cell is used.
+    Translate atoms to the bottom of the cell (vacuum on top).
 
     Args:
-        structure (Structure): The pymatgen Structure object to normalize.
-        use_conventional_cell: Whether to convert to the conventional cell.
+        material (Material): The material object to normalize.
+        vector (List[float]): The vector to translate the atoms by (in crystal coordinates)
+        to (str): The position to translate the atoms to (top, bottom, center)
     Returns:
-        Structure: The normalized pymatgen Structure object.
+        Material: The translated material object.
     """
-    if use_conventional_cell:
-        structure = PymatgenSpacegroupAnalyzer(structure).get_conventional_standard_structure()
-    structure = translate_to_bottom_pymatgen_structure(structure)
-    return structure
+    atoms = to_ase(material)
+    min_z = min(atoms.positions[:, 2])
```

Comment:
We can add `get_max_z` and `get_min_z` to analyze also and act on atoms directly

Reply:
I'm reusing existing functionality

Reply:
??